scripts/cypress_run.py

```diff
@@ -65,7 +64,6 @@ def get_cypress_cmd(
         )
     else:
         # Run local, but split the execution
-        os.environ.pop("CYPRESS_RECORD_KEY", None)
```

Comment:
Do we not want that pop? In the case of running off the repo where the env var is always present, it may always disregard the `use_dashboard` param effectively

Reply:
¯\_(ツ)_/¯ I wasn't sure... I'll add it back in... it certainly doesn't hurt.